docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -40,26 +24,24 @@ networkers=(
 )
 ----
 +
-** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the `Networker` node and its IP address.
+** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the Networker node and its IP address.
+
+.Procedure
 
-. Deploy the other `OpenStackDataPlaneNodeSet` CR. For simplicity, you can
-reuse most of the `nodeTemplate` section from the `openstack` CR designated for
-`Compute` nodes, though strictly speaking, some of the variables may be
-unnecessary because of a more limited set of services running on `Networker`
-nodes. Make sure `edpm_enable_chassis_gw: true` is set to run `ovn-controller`
-in gateway mode.
+. Deploy the `OpenStackDataPlaneNodeSet` CR for your Networker nodes:
 +
-. If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
+[NOTE]
+You can reuse most of the `nodeTemplate` section from the `OpenStackDataPlaneNodeSet` CR that is designated for your `Compute` nodes. However, some of the variables might be unnecessary because of a more limited set of services that are running on `Networker` nodes.
```

Comment:
Just noticed that you have backticks around Networker in this sentence but not elsewhere.